lib/greet.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.greet = function greet(name) {
+  return `hello ${name}`;
+}
+
+if(process.argv[2]) console.log(exports.greet(process.argv[2]));
```

Comment:
You might consider storing the value of process.argv[2] into a variable for user elsewhere. Saves on typing the complex line of code, and also makes it a bit more human readable.